src/static/riot/profiles/user_detail.tag

```diff
@@ -0,0 +1,177 @@
+<user-detail>
+    <h1>User Edit Screen</h1>
```

Comment:
If we do keep the email field here, let's render a human-readable error message in `toastr`. Would also be cool to highlight fields that get rejected by the backend, so the user can look at the field to fix it rather than having to read the fleeting `toastr` message. If you highlight the field, and render an error message next to it, then there's no need for `toastr`, an ideal scenario.
![image](https://user-images.githubusercontent.com/21694127/99607210-2e2b9580-29c0-11eb-8b6f-6ac9e8dc0d5b.png)

Reply:
Definitely a much better look!
![image](https://user-images.githubusercontent.com/16579931/100006235-0c932b00-2d7f-11eb-9228-1ee70368ac38.png)